### PR TITLE
Shows social login email if availiable

### PR DESCRIPTION
**Pull request by belcherj**
#### Changes proposed in this Pull Request

* Shows social connection email if it is available

#### Testing instructions

* Must have api patched
* Be logged in as a non-a8c user
* Goto: /me/security/social-login
* Add social connection
* Refresh
* Observe email address is shown

### CODE REVIEW

**Comment by stephanethomas** [comment]
The code looks good, and works fine.

**Comment by stephanethomas** [comment]
I've added the missing `[Status] Needs Review` label so we can run unit tests against those changes. The server patch requires some changes.

File: client/me/social-login/index.jsx [hunk 1]
```diff
@@ -173,9 +175,12 @@ class SocialLogin extends Component {
 export default connect(
 	state => {
 		const currentUser = getCurrentUser( state );
+		const connections = currentUser.social_login || [];
+		const googleConnection = find( connections, { service: 'google' } );
 
 		return {
-			isUserConnectedToGoogle: currentUser && currentUser.social_signup_service === 'google',
+			socialConnectionEmail: get( googleConnection, 'service_user_email', '' ),
+			isUserConnectedToGoogle: get( googleConnection, 'service', '' ) === 'google',
```
Comment (stephanethomas) [inline client/me/social-login/index.jsx]: It looks like checking we have at least a connection should be enough here:

```javascript
isUserConnectedToGoogle: ! isEmpty( googleConnection ),
```

Comment (belcherj) [reply client/me/social-login/index.jsx]: Resolved

**Pull request merged**